modules/cluster-logging-collector-syslog.adoc

```diff
@@ -0,0 +1,185 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-external.adoc
+
+[id="cluster-logging-collector-syslog_{context}"]
+= Sending logs using the Fluentd *syslog* plug-in (RFC 3164)
+
+You can use the Fluentd *syslog* plug-in to send a copy of your logs to an external syslog server, 
+and not the default Elasticsearch. Note the following about the *syslog* plug-in:
+
+* uses syslog protocol (RFC 3164), not RFC 5424;
+* does not support TLS and thus, is not secure;
+* does not provide Kubernetes metadata, systemd data, or other metadata.
+
+[NOTE]
+====
+This legacy Fluentd *syslog* plug-in method is deprecated and will be removed in a future release.
```

Comment:
Customer and support team are confuse with the word 'legacy', this is  intruduced the first time. Why we call it legacy?
How about: the legacy method using Environment method had been deprecated.

Although we may removed this method in future release too. But I am not sure that is right time.

Reply:
@anpingli The use of the Fluentd plug-ins to forward logs is deprecated and will be removed at some point. @jcantrill Please confirm if this is still the plan. 

Changed to wording to: _The Fluentd *syslog* plug-in method is deprecated..._